tmdb/show.go

```diff
@@ -559,6 +567,31 @@ func (show *Show) SetArt(item *xbmc.ListItem) {
 		}
 	}
 
+	if show.Images != nil && show.Images.Logos != nil {
```

Comment:
I think there are multiple places that have copy-pasted code that is picking available localized image from `[]*Image`. We can move it to a separate method.

Reply:
i did a function that can do all those assigns, but it has a lot of pointers, so i need more time to test it.
for now i made a build without this.

Reply:
done